cilium/cmd/cgroups.go

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import "github.com/spf13/cobra"
+
+// cgroupsCmd represents the cgroups command
+var cgroupsCmd = &cobra.Command{
+	Use:   "cgroups",
+	Short: "Manage cgroup metadata maintained by Cilium",
```

Comment:
Do you see any use case where users will be "managing" cgroup metadata in some way? If not, we should change the description to "Display cgroup metadata...".

Reply:
Well, not right now. I'd just be confused when I see "Display" in the command description, and then suddenly there's a subcommand `list` needed to really display. That's my only concern here, but if you think it's ok this way, I can change the description for the `cgroups` as well.

Reply:
That's a fair point. How about changing the description to "Cgroup metadata"? The list command description in `cgroups_list.go` then expands on the description further.

Reply:
Changed it, thanks for the suggestion.